bin/plugin/commands/changelog.js

```diff
@@ -961,9 +961,9 @@ function getContributorProps( pullRequests ) {
 	}
 
 	return (
-		'## First time contributors' +
+		'## First-time contributors' +
 		'\n\n' +
-		'The following PRs were merged by first time contributors:' +
+		'The following PRs were merged by first-time contributors:' +
```

Comment:
We may also need to update the unit test [snapshot](https://github.com/WordPress/gutenberg/blob/5686e54f36447ed0d38016e15a633ad4a5a05a89/bin/plugin/commands/test/__snapshots__/changelog.js.snap#L287).

```
$ npm run test:unit bin/plugin/commands/test/changelog.js

...
● getContributorProps › verify that the contributors props are properly formatted

    expect(received).toMatchSnapshot()

    Snapshot name: `getContributorProps verify that the contributors props are properly formatted 1`

    - Snapshot  - 2
    + Received  + 2

    - ## First time contributors
    + ## First-time contributors

    - The following PRs were merged by first time contributors:
    + The following PRs were merged by first-time contributors:
...
```

Reply:
Done, thanks.